### PR TITLE
Feat/ssdi pell grant cash assistance

### PR DESCRIPTION
**Pull request by ivonne-hernandez**
ClickUp Ticket: https://app.clickup.com/t/8685fyh8a

What (if any) features are you implementing?
- I added `ssdi` and `pell` grant to the list of `cashAssistanceBenefits`.

What (if anything) did you refactor?
- I had to add `ssdi` and `pell` grant to all of the files that update the `benefits` values via a fetch request, as well as the `Wrapper` and `formData.ts` files.
 
Were there any issues that arose?
- As of right now because our backend has not added these programs, the response is not returning the `ssdi` and `pell` grant properties. 
 
Is there anything that you need from your teammate?
- Can you please add `ssdi` and `pell` grant to the backend? Once it's been added we'll need to QA the response object to ensure that it is posting correctly throughout the app and persisting the values in the `Confirmation` component.

English:
Screenshot 2023-10-16 at 2 45 11 PM
Spanish:
Screenshot 2023-10-16 at 2 54 29 PM
Vietnamese:
Screenshot 2023-10-16 at 2 54 45 PM

`Confirmation` page:
Screenshot 2023-10-16 at 2 44 56 PM

### CODE REVIEW

File: src/Assets/BenefitCategoryLists/cashAssistanceBenefits.tsx [hunk 1]
```diff
@@ -25,6 +25,18 @@ const cashAssistanceBenefits = {
       defaultMessage="State cash assistance for individuals 60 years of age or older (Old Age Pension/OAP)"
     />
   ),
+  ssdi: (
+    <FormattedMessage
+      id="cashAssistanceBenefits.ssdi"
+      defaultMessage="Social security benefit for people with disabilities (Social Security Disability Insurance/SSDI)"
```
Comment (ivonne-hernandez) [inline src/Assets/BenefitCategoryLists/cashAssistanceBenefits.tsx:31]: We will need to add this new FM `id` and `defaultMessage` to the translations UI via api/translations/admin.

File: src/Assets/BenefitCategoryLists/cashAssistanceBenefits.tsx [hunk 2]
```diff
@@ -25,6 +25,18 @@ const cashAssistanceBenefits = {
...
+    />
+  ),
+  pell: (
+    <FormattedMessage
+      id="cashAssistanceBenefits.pell"
+      defaultMessage="Federal grant to finance college costs (Pell Grant)"
```
Comment (ivonne-hernandez) [inline src/Assets/BenefitCategoryLists/cashAssistanceBenefits.tsx:37]: We will need to add this new FM `id` and `defaultMessage` to the translations UI via api/translations/admin.

File: src/Assets/BenefitCategoryLists/cashAssistanceBenefits.tsx [hunk 3]
```diff
@@ -25,6 +25,18 @@ const cashAssistanceBenefits = {
       defaultMessage="State cash assistance for individuals 60 years of age or older (Old Age Pension/OAP)"
     />
   ),
+  ssdi: (
+    <FormattedMessage
+      id="cashAssistanceBenefits.ssdi"
+      defaultMessage="Social security benefit for people with disabilities (Social Security Disability Insurance/SSDI)"
+    />
+  ),
+  pell: (
```
Comment (CalebPena) [inline src/Assets/BenefitCategoryLists/cashAssistanceBenefits.tsx:34]: Shouldn't Pell Grant be in the Child Care, Youth, and Education category?

File: src/Components/SubmitScreen/SubmitScreen.js
Comment (CalebPena) [inline src/Components/SubmitScreen/SubmitScreen.js:1]: I thought we were going to delete this file?

File: src/Assets/updateScreen.js [hunk 4]
```diff
@@ -36,8 +36,10 @@ const getScreensBody = (formData, languageCode) => {
     has_mydenver: formData.benefits.mydenver,
     has_nslp: formData.benefits.nslp,
     has_oap: formData.benefits.oap,
+    has_pell: formData.benefits.pell,
```
Comment (CalebPena) [inline src/Assets/updateScreen.js:39]: I am going to use `has_pell_grant` for the API.

File: src/Components/SubmitScreen/SubmitScreen.js
Comment (ivonne-hernandez) [reply src/Components/SubmitScreen/SubmitScreen.js:1]: I just deleted it.

File: src/Assets/BenefitCategoryLists/childCareBenefits.tsx [hunk 5]
```diff
@@ -25,6 +25,12 @@ const childCareBenefits = {
   upk: (
     <FormattedMessage id="childCareBenefits.univpresc" defaultMessage="Free preschool (Universal Preschool Colorado)" />
   ),
+  pell: (
+    <FormattedMessage
+      id="childCareBenefits.pell"
+      defaultMessage="Federal grant to finance college costs (Pell Grant)"
```
Comment (ivonne-hernandez) [inline src/Assets/BenefitCategoryLists/childCareBenefits.tsx:31]: We will need to add this new FM `id` and `defaultMessage` to the translations UI via api/translations/admin.

File: src/Assets/updateScreen.js (hunk 4 above)
Comment (ivonne-hernandez) [reply src/Assets/updateScreen.js:39]: I've updated the API value to use `has_pell_grant`.

**Review by CalebPena — APPROVED** [review]

**Pull request merged**